src/desktop/components/main_layout/stylesheets/buttons.styl

```diff
@@ -265,14 +263,12 @@
     vertical-align middle
 
 .garamond-button-text
+  text('text')
   display inline-block
   position relative
-  garamond()
-  text-transform uppercase
-  letter-spacing 1px
-  color black
+  color getColor('black100')
   text-decoration none
-  line-height 1
+  line-height 1 !important
```

Comment:
Is this intentionally added? (It seems like the kind of thing that might have slipped through trying to figure out why something wasn't working)

Reply:
Actually intentional, buttons generally have line heights of `1` because they are so never multi-line and sit with the padding better.

Reply:
Oh, and the `!important` is because of the way the media query is specified, it makes this a lil hard to override (would have to include the breakpoint as well)... In practice, with the `Text` component this isn't an issue, but it is here 🤷‍♂️